### ISSUE TITLE
Every second behavior

### ISSUE THREAD

**Issue by xpepermint** [harrisiirak/cron-parser#51]
```js
var now = new Date(); 
var cron = require('cron-parser'); 
var s = cron.parseExpression('* * * * * *', {currentDate: now, iterator: true}); 
console.log(`now: ${now.toString()}, next: ${s.next().value}`);
// -> now: Tue Dec 15 2015 10:11:52 GMT+0100 (CET), next: Tue Dec 15 2015 10:11:52 GMT+0100 (CET)
```

Is this expected result (the two dates are the same)? I would expect the `next` to be 1sec in the future.

**Comment by xpepermint** [comment]
great!

**Comment by harrisiirak** [comment]
Np! Thank you for reporting that one. It's really weird that I didn't notice that before, because it's quite obvious as I see it now. Released as "1.1.0".
